src/main/java/com/netuitive/ananke/statsd/client/StatsDClient.java

```diff
@@ -0,0 +1,36 @@
+package com.netuitive.ananke.statsd.client;
+
+import com.netuitive.ananke.statsd.client.request.DecrementRequest;
+import com.netuitive.ananke.statsd.client.request.EventRequest;
+import com.netuitive.ananke.statsd.client.request.GaugeRequest;
+import com.netuitive.ananke.statsd.client.request.HistogramRequest;
+import com.netuitive.ananke.statsd.client.request.IncrementRequest;
+import com.netuitive.ananke.statsd.client.request.ServiceCheckRequest;
+import com.netuitive.ananke.statsd.client.request.SetRequest;
+import com.netuitive.ananke.statsd.client.request.TimedRequest;
+import com.netuitive.ananke.statsd.client.request.TimingRequest;
+import java.io.IOException;
+
+/**
+ *
+ * @author john.king
+ */
+public interface StatsDClient {
+    public void decrement(DecrementRequest req) throws IOException;
+    
+    public void event(EventRequest req) throws IOException;
+    
+    public void gauge(GaugeRequest req) throws IOException;
```

Comment:
I would also like to see some convenient overload method for each type of metric so that user doesn't have to construct the Request for simple use cases, e.g. gauge(String metric, Long value).